.gitlab-ci.yml

```diff
@@ -77,6 +77,22 @@ gcp-dev:
     - rake clobber
     - rake configure_serviceaccount_ci_restore
     - rake
+    # Run smoke tests for Preferences
+    - rake sh['sh -c "
```

Comment:
This command is pretty complex, and is something that we might want to run ourselves. Can you make two new rake tasks, `:smoke_test_preferences` and `:smoke_test_flowmanager` (or something), and call them from here?

Reply:
Makes sense!